change/@fluentui-react-21171906-3dee-475e-afba-8e2349b7cbdd.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "react-button: Removing last remnants of legacy patterns (#18519)",
```

This goes in the release notes, so be sure the message is correct
```suggestion
  "comment": "SwatchColorPicker: Updated colorCell style to fix vertical alignment bug",
```